packages/webapp/src/components/Animals/Inventory/index.tsx

```diff
@@ -14,44 +14,95 @@
  */
 import Table from '../../../components/Table';
 import Layout from '../../../components/Layout';
+import PureSearchBarWithBackdrop from '../../PopupFilter/PureSearchWithBackdrop';
+import NoSearchResults from '../../../components/Card/NoSearchResults';
 import type { AnimalInventory } from '../../../containers/Animals/Inventory/useAnimalInventory';
 import { TableV2Column, TableKind } from '../../Table/types';
-import type { DefaultTheme } from '@mui/styles';
+import type { Dispatch, SetStateAction } from 'react';
+import styles from './styles.module.scss';
+import clsx from 'clsx';
+
+export type SearchProps = {
+  searchString: string | null | undefined;
+  setSearchString: Dispatch<SetStateAction<string[]>>;
+  placeHolderText: string;
+  searchResultsText: string;
+};
 
 const PureAnimalInventory = ({
-  tableData,
+  filteredInventory,
   animalsColumns,
-  theme,
-  isMobile,
+  zIndexBase,
+  backgroundColor,
+  isDesktop,
+  searchProps,
 }: {
-  tableData: AnimalInventory[];
+  filteredInventory: AnimalInventory[];
   animalsColumns: TableV2Column[];
-  theme: DefaultTheme;
-  isMobile: boolean;
+  zIndexBase: number;
+  backgroundColor: string;
+  isDesktop: boolean;
+  searchProps: SearchProps;
 }) => {
+  const { searchString, setSearchString, placeHolderText, searchResultsText } = searchProps;
+  const hasSearchResults = filteredInventory.length !== 0;
+
   return (
     <Layout
       classes={{
         container: {
-          backgroundColor: theme.palette.background.paper,
-          borderRadius: '8px',
-          border: '1px solid var(--Colors-Primary-Primary-teal-50)',
-          marginTop: '16px',
+          backgroundColor: backgroundColor,
+          borderRadius: isDesktop && '8px',
+          border: isDesktop && '1px solid var(--Colors-Primary-Primary-teal-50)',
+          marginTop: isDesktop && '16px',
+          padding: !isDesktop ? '0px' : '16px',
         },
       }}
       hasWhiteBackground
       footer={false}
     >
-      <Table
-        kind={TableKind.V2}
-        alternatingRowColor={true}
-        columns={animalsColumns}
-        data={tableData}
-        shouldFixTableLayout={!isMobile}
-        minRows={tableData.length}
-        dense={false}
-        showHeader={!isMobile}
-      />
+      <div
+        className={clsx(
+          isDesktop ? styles.SearchAndFilter_container___desktop : styles.SearchAndFilter_container,
```

Comment:
I like BEM overall, but since we haven't made a decision to go for that throughout the app, I'd advocate for consistency here and naming the styles more similarly to how classes are currently named in other files. In general I'm all for BEM but I think these will be out of place unless there's overall consensus to go with BEM from now on. If we do make a decision to go with BEM from now on, I think the convention would actually be something like `search-and-filter__container--desktop`. Unless this is a different one and not BEM?

Reply:
I think there are some react problems with the - or something. I linked the article I used to try to apply it but it may be out of date. 

I just added it to discuss the isDesktop inline breakpoints readability.